frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/ApprovalTaskNode.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+import { observer, Node, NodeModel, useHover, createSvgIdUrl } from '@patternfly/react-topology';
+import * as cx from 'classnames';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom-v5-compat';
+import { K8sResourceKind, WatchK8sResults } from '@console/dynamic-plugin-sdk/src/lib-core';
+import { resourcePathFromModel, truncateMiddle } from '@console/internal/components/utils';
+import { useK8sWatchResources } from '@console/internal/components/utils/k8s-watch-hook';
+import { referenceForModel } from '@console/internal/module/k8s';
```

Comment:
avoid using `referenceForModel` as it is deprecated. Use `getGroupVersionKindForModel` as I suggested below.